iothub_client/src/iothubtransport_mqtt_common.c

```diff
@@ -1203,15 +1203,44 @@ static int subscribeToNotifyStateIfNeeded(PMQTTTRANSPORT_HANDLE_DATA transport_d
     return result;
 }
 
+// The tokenData variable carries a string in the format "<name>=<value".
```

Comment:
"\<name\>=\<value" to "\<name\>=\<value\>"

Reply:
Good catch. However this function has been removed.